### PR TITLE
fix: dimension min max column only for hgv and trl

### PR DESCRIPTION
**Pull request by m-ujaffer**
a few fixes in weights-psv and id updates. Also added missing unit tests for notes component.

### CODE REVIEW

File: src/app/technical-record/notes/notes.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  it('should create view only for psv correctly populated', () => {
+    component.activeRecord = TESTING_UTILS.mockTechRecord({
+      remarks: 'some remarks',
+      vehicleType: 'psv'
+    });
+
+    fixture.detectChanges();
+
+    expect(component).toBeDefined();
```
Comment (alleypa) [inline src/app/technical-record/notes/notes.component.spec.ts:45]: lets remove this

File: src/app/technical-record/notes/notes.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      vehicleType: 'hgv'
+    });
+
+    fixture.detectChanges();
+
+    expect(component).toBeDefined();
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should create view only for psv correctly populated', () => {
```
Comment (alleypa) [inline src/app/technical-record/notes/notes.component.spec.ts:37]: `should create view only for psv populated data`

File: src/app/technical-record/notes/notes.component.spec.ts (hunk 1 above)
Comment (m-ujaffer) [reply src/app/technical-record/notes/notes.component.spec.ts:45]: fixed

File: src/app/technical-record/notes/notes.component.spec.ts (hunk 2 above)
Comment (m-ujaffer) [reply src/app/technical-record/notes/notes.component.spec.ts:37]: fixed

**Pull request merged**